app/src/test/java/tech/relaycorp/gateway/pdc/local/routes/ParcelCollectionRouteTest.kt

```diff
@@ -111,4 +139,57 @@ class ParcelCollectionRouteTest {
                 }
             }
         }
+
+    @Test
+    fun `Server should ignore binary acks`() =
+        runBlockingTest {
+            testPDCServerRoute(route) {
+                handleWebSocketConversation(ParcelCollectionRoute.URL_PATH) { _, outgoing ->
+                    outgoing.send(Frame.Binary(true, "abc".toByteArray()))
+
+                    verify(collectParcels, never()).processParcelAck(any())
```

Comment:
I think this may be happening too soon: `collectParcels.processParcelAck()` might actually be called (if there's a bug), it just might not be called when this `verify()` is done. I'd suggest to wait until the connection is closed before doing this assertion.